Ticket/Index.php

```diff
@@ -16,33 +16,45 @@
 	<?php if (isset($userLoggedIn)) 
 	{ if(isset($_GET['ticketId']) && ticketExistance($_GET['ticketId'])) 
 		{ $ticketId = $_GET['ticketId']; ?>
+
+	<div id="ticketActions"></div>
 	
-	<div ID="ticketContainer">
 
-			<div ID="ticketDetails">
-			<?php include("details.php"); ?>	
-    		</div>
+		<div class="row no-gutters">
+			<div id="info" class="col-sm-12 col-md-3">
+				<div id="ticketPeople">
+					<?php include("people.php"); ?>
+				</div>
+				<div id="ticketDates">
+					<?php include("dates.php"); ?>
+				</div>
+			</div>
 
-    		<div ID="ticketPeople">
-			<?php include("people.php"); ?>	
-    		</div>
+			<div id="main" class="col-sm-12 col-md-9">
+				<div class="row no-gutters">
+					<div id="ticketDetails">
+						<?php include("details.php"); ?>
+					</div>
+				</div>
 
-	</div>  
-		  <div ID="ticketCreate">
-		  <?php include("createComment.php"); ?>	
-		  </div>
-		  
-		  <div ID="ticketComments">
-		  <?php include("viewComments.php"); ?>	
-		  </div>
+					<div class="row no-gutters mt-3">
+						<div id="ticketCreate">
+							<?php include("createComment.php"); ?>	
+						</div>
+					</div>
 
-		  <?php include("../Global/editUserModal.php"); ?>
-		  <?php include("ticketModal.php"); ?>
- 
-<?php } else { echo "<p id='loginMessage'> Ticket ID is invalid! </p>"; }
-	} else { echo "<p id='loginMessage'> You need to login to access this page </p>"; } 
-	?>
+					<div class="row no-gutters mt-3">
+						<div id="ticketComments">
+		  					<?php include("viewComments.php"); ?>	
+		  				</div>
+					</div>
+			</div>
+		</div>
+	
+		<?php include("../Global/editUserModal.php"); ?>
+		<?php include("ticketModal.php"); ?>
+		<?php } else { echo "<p id='loginMessage'> Ticket ID is invalid! </p>"; }
+			} else { echo "<p id='loginMessage'> You need to login to access this page </p>"; } 
```

Comment:
Get rid of the you need to login to access this page else statement, got another function that deals with it so it is no longer needed